src/sql/azdata.proposed.d.ts

```diff
@@ -194,4 +194,23 @@ declare module 'azdata' {
 
 	export interface ImageComponentProperties extends ComponentProperties, ComponentWithIconProperties {
 	}
+
+	export interface InputBoxProperties extends ComponentProperties {
+		value?: string;
+		ariaLive?: string;
+		placeHolder?: string;
+		inputType?: InputBoxInputType;
+		required?: boolean;
+		multiline?: boolean;
+		rows?: number;
+		columns?: number;
+		min?: number;
+		max?: number;
+		/**
+		 * Whether to stop key event propagation when enter is pressed in the input box. Leaving this as false
+		 * means the event will propagate up to any parents that have handlers (such as validate on Dialogs)
+		 */
+		stopEnterPropagation?: boolean;
+		validationErrorMessage?: string;
```

Comment:
You don't need to copy the entire definition - it'll mix them in at runtime so you can just define it like this

```
export interface InputBoxProperties {
    validationErrorMessage?: string;
}
```